editor/filesystem_dock.cpp

```diff
@@ -1983,11 +1986,9 @@ FileSystemDock::FileSystemDock(EditorNode *p_editor) {
 
 */
 	file_options = memnew(PopupMenu);
-	file_options->set_hide_on_window_lose_focus(true);
```

Comment:
What's the rationale for changing this?

Reply:
I don't remember removing this, will rollback to master.